ietf-optical-impairment-topology.yang

```diff
@@ -55,66 +55,67 @@ module ietf-optical-impairment-topology {
      Provisions Relating to IETF Documents
      (http://trustee.ietf.org/license-info).";
 
-  revision 2019-05-30 {
+  revision 2020-03-03 {

```

Comment:
Shall we use 2020-03-09 as revision date? This will be the submission date of the draft.

Reply:
yes, modified with new commit